src/main/java/com/doyak/reflector/domain/Post.java

```diff
@@ -0,0 +1,71 @@
+package com.doyak.reflector.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.doyak.reflector.domain.common.BaseEntity;
+import com.doyak.reflector.domain.enums.Site;
+
+import jakarta.persistence.CascadeType;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.Lob;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.OneToMany;
+import lombok.Getter;
+
+@Entity
+@Getter
+public class Post extends BaseEntity{
```

Comment:
```suggestion
@Builder
@NoArgsConstructor(access = AccessLevel.PROTECTED)
@AllArgsConstructor
@Table(name = "posts")
public class Post extends BaseEntity{
```
`Post.java` 와 `Block.java` 엔티티 객체 생성 시 생성자를 직접 호출하기보다는 `@Builder` 어노테이션을 사용하여 생성하는 편이 유지보수성과 안정성 측면에서 더 유리할 것 같습니다! (빌더 사용을 위해서는 `@AllArgsConstructor`가 필요합니다.) 또한 JPA는 객체 생성 시 내부적으로 기본 생성자를 필요로 하기 때문에, 해당 요구사항을 충족하기 위해 `@NoArgsConstructor(access = AccessLevel.PROTECTED)`를 추가해주면 좋을 것 같습니다.

테이블명은 PostgreSQL 관례상 복수형 사용을 권장하므로, 모든 테이블을 복수형으로 통일하여 일관성을 주는 것이 좋을 것 같아요! 이 부분은 지난 회의에서 말씀드리려고 했는데 제가 잊어버렸네요. 죄송합니다...

Reply:
좋은 지적 감사합니다! 이렇게 하는게 Post객체를 더 안전하게 객체 생성하는 것 같아요. 이 방향으로 수정 진행하도록 하겠습니다.
그런데 Block객체는 이후 Text, Code 블럭이 상속할 abstract 클래스라서 builder 어노테이션은 구체 클래스에 붙여서 생성하도록 하겠습니다!